workspaces/rbac/plugins/rbac/src/components/CreateRole/MembersDropdownOption.tsx

```diff
@@ -26,46 +25,47 @@ import { SelectedMember } from './types';
 type MembersDropdownOptionProps = {
   option: SelectedMember;
   state: AutocompleteRenderOptionState;
+  props: React.HTMLAttributes<HTMLLIElement>;
 };
 
-const useStyles = makeStyles(theme => ({
-  optionLabel: {
-    color: theme.palette.text.primary,
-  },
-  optionDescription: {
-    color: theme.palette.text.secondary,
-  },
-}));
-
 export const MembersDropdownOption = ({
+  props,
   option,
   state,
 }: MembersDropdownOptionProps) => {
-  const classes = useStyles();
   const { inputValue } = state;
   const { label, etag } = option;
   const matches = match(label, inputValue, { insideWords: true });
   const parts = parse(label, matches);
 
   return (
-    <Box key={`${etag}`}>
-      {parts.map(part => (
-        <Typography
-          key={`${part.text}-${etag}`}
-          component="span"
-          className={classes.optionLabel}
-          sx={{
-            fontWeight: part.highlight ? 400 : 700,
-          }}
-          data-testid={option.label}
-        >
-          {part.text}
-        </Typography>
-      ))}
-      <br />
-      <Typography className={classes.optionDescription}>
+    <li
+      {...props}
+      key={`${etag}`}
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'flex-start',
+      }}
```

Comment:
Isn't this the default when using block elements?

Maybe you can remove this custom style by using a div instead of a span below?

Reply:
I got that, but its only a span because you define in the code `component="span"`. So removing this 5 lines + the component=span code should result in the same. You maybe can check that later. Fine for now.